apps/teams-test-app/src/components/RememberScrollToggle.tsx

```diff
@@ -0,0 +1,54 @@
+import throttle from 'lodash.throttle';
```

Comment:
What's the difference between using throttle and wait if you don't mind me asking? Just genuinely curious

Reply:
Not sure what `wait` function you mean, but `throttle` will make sure that a function only gets called once every X milliseconds (2000 in this case). Will prevent a function from being called on every scroll event for example.